packages/instantsearch.js/src/middlewares/createInsightsMiddleware.ts

```diff
@@ -136,12 +136,26 @@ export function createInsightsMiddleware<
       // Otherwise, the `init` call might override it with anonymous user token.
       userTokenBeforeInit = userToken;
     });
-    insightsClient('init', {
-      appId,
-      apiKey,
-      useCookie: true,
-      ...insightsInitParams,
-    });
+
+    const version = insightsClient.version;
+    let canInit = false;
+    if (typeof version !== 'undefined') {
```

Comment:
sorry, I think this is a breaking change, as the current version of the plugin doesn't check. Maybe if we add && $internal?

Reply:
Indeed, I forgot to based the condition on the `$$internal` flag, thanks!